### ISSUE TITLE
Vim .swp files cause false positive for acf-json sync

### ISSUE THREAD

**Issue by johnmenick** [elliotcondon/acf#613]
To reproduce using acf-json with WP already synced:

1. Open one of the JSON files in vim, causing a .swp file to be created
2. Reload ACF field group page
3. "Sync available" count will uptick by 1 and showing the json file minus the .swp extension
4. If you attempt to sync, an empty group will be created.

**Comment by elliotcondon** [comment]
Hi @johnmenick 

Thanks for the bug report. What version of ACF PRO are you using?
The ACF JSON class should only be including .json files thatks to this line:
```
if( pathinfo($file, PATHINFO_EXTENSION) !== 'json' ) continue;
```

Perhaps this is incorrectly allowing a .swp file? I'm not too familiar with .swp

**Comment by johnmenick** [comment]
We are using version 5.5.3. Is this version not available in this repo? That line of code does not appear in our version.

**Comment by elliotcondon** [comment]
Hi @johnmenick 

ACF PRO is a premium plugin. Please update by activating your license key in the 'Custom Fields -> Updates' page

**Comment by johnmenick** [comment]
Your statement doesn't address the issue. So what is the latest pro version number? Does the update fix the problem? 

I'm not a PHP dev, but I'm fairly sure your problem is here line 128 in /core/json.php:

```php
if( strpos($file, '.json') === false ) {
```

That doesn't check the extension, only the presence of the string ".json."

**Comment by elliotcondon** [comment]
Hi @johnmenick 

The fix was introduced in ACF PRO 5.5.10.

That line now reads:
```
if( pathinfo($file, PATHINFO_EXTENSION) !== 'json' ) continue;
```
